symbols/arm7.yml

```diff
@@ -56,6 +56,16 @@ arm7:
         Once the memory mapping has been completed, a constant containing the address to NitroSpMain is loaded into a register (r1), and a `bx` branch will jump to NitroSpMain.
         
         No params.
+    - name: do_autoload
+      address:
+        EU: 0x2380118
+        NA: 0x2380118
+        JP: 0x2380118
+    - name: StartAutoloadDoneCallback
+      address:
+        EU: 0x2380188
+        NA: 0x2380188
+        JP: 0x2380188
```

Comment:
Are these also different than the ARM9 versions? If so, we should also tag these with an arm7 suffix in pmdsky-debug.

Reply:
Done.